src/core/math/line2/intersectToLine.js

```diff
@@ -0,0 +1,19 @@
+const vec2 = require('../vec2')
+const { solve2Linear } = require('../../utils/various')
+
+/**
+ * Return the point of intersection between the given lines.
+ *
+ * The point will have Infinity values if the lines are paralell.
+ * The point will have NaN values if the lines are the same.
+ *
+ * @param {line2} line1 a 2D line for reference
+ * @param {line2} line2 a 2D line for reference
+ * @return {vec2} the point of intersection
+ */
+const intersectToLine = (line1, line2) => {
```

Comment:
sorry but the name of this function does not make sense, could be something like:
- intersectionPointBetweenLines
- getIntersectionBetweenLines

Reply:
I moved the V1 Line2 and Line3 code into 'moveOutOfCode' directory.

Name changes... Actually, a bunch of name changes should be made.
Are we following the JavaScript naming conventions as much as possible?

Reply:
How are these?

line2...

intersectToLine => intersectPointOfLine

line3...

intersectToPlane => intersectPointOfPlane